configuration.yaml

```diff
@@ -8,12 +8,7 @@ global:
 cancerBiomarkers:
   inputBucket: gs://otar000-evidence_input/CancerBiomarkers/data_files
   outputBucket: gs://otar000-evidence_input/CancerBiomarkers/json
-  # Downloaded separately using wget, because FTP.RemoteProvider cannot handle recursive directory downloads.
-  drugIndex: ftp.ebi.ac.uk/pub/databases/opentargets/platform/22.02/output/etl/parquet/diseases
-ChEMBL:
-  evidence: gs://otar008-chembl/cttv008-20-01-2022.json.gz
-  stopReasonCategories: gs://otar000-evidence_input/ChEMBL/data_files/chembl_predictions-2022-01-24.tsv
-  outputBucket: gs://otar000-evidence_input/ChEMBL/json
+  drugIndex: gs://open-targets-pre-data-releases/22.04/output/etl/parquet/molecule
```

Comment:
I see there's an other section of ChEMBL. But why are you deleting this one and keeping the one, which is out of the alphabetic order?

Reply:
Fixed in https://github.com/opentargets/evidence_datasource_parsers/pull/125/commits/689142fa8dc1ebd6be9d24b9b41bdb4e5ea5ed4f.